models/baseModels/Invoice/Invoice.ts

```diff
@@ -138,12 +144,17 @@ export abstract class Invoice extends Transactional {
       await payment?.submit();
       await this.load();
     }
+
+    await this._updateReturnInvoiceOutStanding();
+    await updateReturnCompleteStatus(this);
   }
 
   async afterCancel() {
     await super.afterCancel();
     await this._cancelPayments();
     await this._updatePartyOutStanding();
+    await this._updateReturnInvoiceOutStanding();
+    await this._updateReturnCompleteStatus();
```

Comment:
Why is there an instance method and an external function both with the same name?